app/pages/project/talk-images.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import getSubjectLocation from '../../lib/get-subject-location.coffee';
+import TalkStatus from './talk-status';
+import Thumbnail from '../../components/thumbnail';
+
+export default class TalkImages extends React.Component {
+
+  render() {
+    return (
+      <div className="project-home-page__container">
+
+        {this.props.images.map((image) => {
+          const subject = getSubjectLocation(image);
+          return (
+            <div key={subject.id} className="project-home-page__talk-image">
+              <Thumbnail alt="" src={subject.src} format={subject.format} controls={false} />
```

Comment:
I think you actually want something more like https://thumbnails.zooniverse.org/300x300/panoptes-uploads.zooniverse.org/production/subject_location/fb97e72f-19e2-4f9a-af0b-28a7d1544d76.jpeg so maybe `width=300 height=300`.